moveit_ros/planning/plan_execution/include/moveit/plan_execution/plan_execution.h

```diff
@@ -90,7 +90,8 @@ class PlanExecution
   };
 
   PlanExecution(const planning_scene_monitor::PlanningSceneMonitorPtr& planning_scene_monitor,
-                const trajectory_execution_manager::TrajectoryExecutionManagerPtr& trajectory_execution);
+                const trajectory_execution_manager::TrajectoryExecutionManagerPtr& trajectory_execution,
+                const std::shared_ptr<rclcpp::Node> node);
```

Comment:
Move node to first position to keep it consistent with the other classes. Also make it const reference.